src/components/WebexMemberRoster/WebexMemberRoster.scss

```diff
@@ -4,4 +4,16 @@
     color: $md-gray-40;
     border-bottom: 0.1rem solid $md-gray-30;
   }
+
+  .outside-warning {
+    color: rgb(217, 127, 0);
```

Comment:
Let's follow the specification for [colors from Momentum](https://momentum.design/tokens/color), Webex design system.
Add a [new Sass variable](https://github.com/webex/components/blob/master/src/styles/_variables.scss#L22) `$md-yellow-40`

Here's the yellow gradient from momentum:
![image](https://user-images.githubusercontent.com/6610308/113062929-59da3b80-9169-11eb-9263-1bdddb5edb74.png)